### DISCUSSION TITLE
Attention: tập trung -> chú ý

### DISCUSSION THREAD

**Discussion by lkhphuc** [d2l-ai/d2l-vi#2745] [Terminology]
> @LongNguyen1984  Chữ "tập trung" mình thấy cũng rất khó hiểu trong tiếng Việt, khi đọc dễ liên tưởng là tập hợp một nhóm nào đó, liệu ta có nên chuyển bằng cụm từ "tạo chú ý", mọi người thấy sao?

Từ này dịch là `tập trung` mình cũng có hơi gợn gợn lấn cấn, sẵn tiện có bác @LongNguyen1984 cũng nói nên mình đưa ra đây thảo luận. Từ này lúc đầu tiên đưa vào glossary mình cũng có tham gia thảo luận mà không nhớ tại sao không đề xuất là `chú ý`.

Từ `chú ý` đi riêng hầu như không thể nhầm sang từ nào khác, mà từ `tập trung` thì còn có nghĩa là `tụ họp / gather /concentrate`.

Từ này đã được sử dụng khá nhiều trong sách, nếu sửa lại thì sẽ cần khá nhiều công sức, và có thể làm bối rối vài bạn đọc. Tuy nhiên trước đây có từ `recurrent` cũng được sửa lại sau khi được sử dụng khá phổ biến.

**Comment by thanhcsf** [comment]
> Từ này đã được sử dụng khá nhiều trong sách, nếu sửa lại thì sẽ cần khá nhiều công sức, và có thể làm bối rối vài bạn đọc. Tuy nhiên trước đây có từ `recurrent` cũng được sửa lại sau khi được sử dụng khá phổ biến.

Anh em cứ thảo luận thôi nhé, mình sẽ tiến hành điều chỉnh sau. Khi có bản cập nhật mới thì ta sẽ thông báo với anh Tiệp để truyền thông cho việc đó. Việc công bố cuốn chiếu trong khi còn hoàn thiện bản thảo thì ta đành chấp nhận có các sự thay đổi như thế này, âu cũng tốt mà.

**Comment by rootonchair** [comment]
Em không đồng tình lắm, mọi người có thể đưa ra vài case không ạ?
